docs/CreateEKSClusterForScalarDB.md

```diff
@@ -0,0 +1,139 @@
+# Create an EKS cluster for ScalarDB Server
+
+This document explains how to create an EKS cluster for the ScalarDB Server deployment. For details on how to deploy ScalarDB Server on the EKS cluster, please see [Deploy ScalarDB Server on Amazon EKS (Amazon Elastic Kubernetes Service)](./ManualDeploymentGuideScalarDBServerOnEKS.md).
+
+## Steps
+
+You must create an EKS cluster based on the following requirements, recommendations, and your project's requirements. For details on how to create the EKS cluster, please see the [AWS official document](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+
+## Requirements
+
+There is one requirement for the deployment of ScalarDB Server. Please configure your EKS cluster based on the following requirement and your project's requirements.
+
+* You must create the EKS cluster with Kubernetes version 1.21 or higher for the ScalarDB Server deployment.
+
+## Recommendations (Optional / Not required)
+
+There are some recommendations for the deployment of ScalarDB Server. These are not required. So, you can choose to apply these recommendations or not based on your requirement.
+
+### Use 4vCPU / 8GB memory nodes for the worker node in the ScalarDB Server node group
+
+From the perspective of commercial license, resources for one pod of ScalarDB Server are limited to 2vCPU / 4GB memory. Also, it is recommended to deploy one ScalarDB Server pod and one Envoy pod on one worker node.
+
+In other words, the following components run on the one worker node.
+
+* ScalarDB Server pod (2vCPU / 4GB)
+* Envoy proxy (0.2 ~ 0.3 vCPU / 256 ~ 328 MB)
+* Kubernetes components
+
+So, you should use the worker node that has 4vCPU / 8GB memory resources. It is recommended to run only the above components on the worker node for ScalarDB Server. However, if you want to run other pods on the worker node for ScalarDB Server, you should use the worker node that has over 4vCPU / 8GB memory.
+
+Note that you should configure resource limits based on your system's workload if the Envoy pod exceeds the above resource usage. Also, you should consider scaling out the worker node and the ScalarDB Server pod if all the ScalarDB Server pod exceeds the above resource usage and the latency is high (throughput is low) in your system.
+
+### Create a node group for other application pods than ScalarDB Server pods
+
+It is recommended to run only pods related to ScalarDB Server on the worker node for ScalarDB Server. If you want to run other application pods on the same EKS cluster, you should create other node groups for your application pods.
+
+### Create a node group for monitoring components (kube-prometheus-stack and loki-stack)
```

Comment:
@josh-wong  
Yes. We can use `node group` based on the Amazon official document in our EKS documents.

(In AKS documents, we use `node pool` instead of `node group` because Azure official document uses `node pool`.)

Reply:
Got it. Thanks!